src/sdl/index.ts

```diff
@@ -105,6 +107,28 @@ export class SDL {
     }
   }
 
+  static validateStorage(name: string, storage: v2ResourceStorage | v2ResourceStorageArray | undefined) {
+    if (!storage) {
+      throw new Error("Storage is required for service " + name);
+    }
+
+    const storages = isArray(storage) ? storage : [storage];
```

Comment:
unrelated to PR. Not the first time a usual op used. Can we add some lib like `lodash` it would help around this and other projects

Reply:
We could and we have lodash in the Cloudmos repo, but in this case, the lodash `isArray` is literally `Array.isArray` so might not be necessary.
![image](https://github.com/akash-network/akashjs/assets/15185355/ac497c05-b4ff-44f9-9c8e-20efa7780524)

Reply:
I meant castAtray lodash fn in this case. But great to know there's no objection